Runtime/C#/Runtime/BebopWriteResult.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bebop.Runtime
+{
+    /// <summary>
+    /// Details the result of attempting to encode a record to a buffer
+    /// </summary>
+    public readonly struct BebopWriteResult
```

Comment:
Because the method is `EncodeIntoBuffer` this struct should be called `BebopEncodeResult`

Reply:
Ended up deleting this struct.